provisioning/service/src/Contract/SDKUtils.cs

```diff
@@ -5,7 +5,7 @@ namespace Microsoft.Azure.Devices.Provisioning.Service
 {
     internal class SDKUtils
     {
-        private const string ApiVersionProvisioningPreview = "2017-11-15";
+        private const string ApiVersionProvisioningPreview = "2018-04-01";
```

Comment:
Please rename - we probably forgot to update the variable name: `ApiVersion` is what we use in `ClientApiVersionHelper.cs` in the IoT Hub SDK.